src/main/java/tornadofx/Builders.kt

```diff
@@ -109,6 +109,12 @@ fun Pane.progressbar(property: Property<Double>, op: (ProgressBar.() -> Unit)? =
     progressProperty().bind(property)
 }
 
+fun Pane.slider(min: Double? = null, max: Double? = null, value: Double? = null, op: (Slider.() -> Unit)? = null) = opcr(this, Slider().apply {
```

Comment:
I thought about that, but Slider has a lot of magic numbers in it for defaults, so if they ever changed we'd have to keep ours up to date.  I think the way I did it is probably the best way to do it as it will only create the properties it needs, and they have way more overhead than the `adjustValues()` calls.

I think I like it how it is now (unless you think I should remove the orientation param).